v0/destinations/google_cloud_function/util.js

```diff
@@ -0,0 +1,63 @@
+const {
+    defaultBatchRequestConfig
+} = require("../../util");
+
+const {
+    TRIGGERTYPE
+} = require("./config");
+
+function generateBatchedPayloadForArray(events) {
+    const batchResponseList = [];
+    const metadata = [];
+
+    // extracting destination
+    // from the first event in a batch
+    const {
+        destination
+    } = events[0];
+    const Config = destination.Config;
+    const {
+        googleCloudFunctionUrl,
+        triggerType,
+        apiKeyId,
+        gcloudAuthorization
+    } = Config;
```

Comment:
You can directly use destination.Config here only. No need for line 18

Reply:
fixed